src/components/Charts/data.ts

```diff
@@ -0,0 +1,3048 @@
+import { ChartEntry, PriceChartEntry, TokenData } from "./types";
+
+export const priceData: PriceChartEntry[] = [
+  {
+    time: Math.floor(+new Date("2018-10-19") / 1000),
+    open: 180.34,
+    high: 180.99,
+    low: 178.57,
+    close: 179.85,
+  },
+  {
+    time: Math.floor(+new Date("2018-10-22") / 1000),
+    open: 180.82,
+    high: 181.4,
```

Comment:
Why do we use random data for some charts and predefined for some?

Reply:
because for the line chart, it will look good if we use random numbers,
When I do random for candle charts, there are so many big gaps between the candle, so I decided to use predefined data

Reply:
I think we should use random data, this is over 3000 lines which increases project size unnecessarily